tests/aws/services/lambda_/test_lambda.py

```diff
@@ -696,6 +697,66 @@ def assert_events():
         )
         snapshot.match("invoke-result-read-number-after-timeout", result)
 
+    @markers.aws.validated
+    @markers.snapshot.skip_snapshot_verify(
```

Comment:
Should we group them similar to `tests.aws.services.lambda_.test_lambda_common.TestLambdaRuntimesCommon.test_introspection_invoke`?
It would be helpful to see what's actionable and what's simply LocalStack extras. For some, I guess we don't care.